lib/output.js

```diff
@@ -44,7 +44,7 @@ export default class Output {
         this.data = data;
       }
     }
-    if (lessThan(this.value, BigInt(1))) {
+    if (!this.isNFT() && !this.isNST() && lessThan(this.value, BigInt(1))) {
```

Comment:
So NFTs and NSTs are allowed to have no only 0 but also negative value?

Reply:
i guess yes 🤔 the id can be any value.